versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -52,6 +52,24 @@ spec:
       description:
         description: Short description of field
         type: string
+      metric_type:
+        description: Metric type
+        type: string
+        enum:
+        - counter
+        - gauge
+      unit:
+        description: Unit
+        type: string
+        enum:
+        - byte
```

Comment:
I know `byte` is here because ES supports it but it's the same as `b`, right? To avoid confusion should we remove one of them from the allowed values?

Reply:
arg actually I copied over the byte units instead of time units 🤦 will fix it soon. There was no consensus around using byte units here, so we stick to just `byte` for now

Reply:
Ha, I didn't even notice that 😊 .